src/cbmc/cbmc_parse_options.cpp

```diff
@@ -389,6 +389,13 @@ void cbmc_parse_optionst::get_command_line_options(optionst &options)
   else
     options.set_option("sat-preprocessor", true);
 
+  if (cmdline.isset("random-seed"))
+    options.set_option("random-seed", cmdline.get_values("random-seed"));
+
+  if (cmdline.isset("random-var-freq"))
+    options.set_option(
+      "random-var-freq", cmdline.get_values("random-var-freq"));
+
```

Comment:
These could have default values set here.  This is especially useful for random-seeds.